delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/storages/operations/s3/S3FileOperations.scala

```diff
@@ -67,7 +67,7 @@ object S3FileOperations {
                 )
                 .compile
```

Comment:
Should not this go away no ?

Reply:
I've removed the compile, test will arrive soon